Sources/TokamakCore/Fiber/Fiber.swift

```diff
@@ -134,7 +144,14 @@
       if let element = element {
         self.element = element
       } else if Renderer.isPrimitive(view) {
-        self.element = .init(from: .init(from: view))
+        self
+          .element =
+          .init(from: .init(from: view, shouldLayout: reconciler?.renderer.shouldLayout ?? false))
```

Comment:
nit: weird formatting?
```suggestion
        self.element =
          .init(from: .init(from: view, shouldLayout: reconciler?.renderer.shouldLayout ?? false))
```